This patch is part of a series of patches for festival for
Debian GNU/Linux, to interoperate with Debian sgml

Index: b/lib/sable-mode.scm
===================================================================
--- a/lib/sable-mode.scm
+++ b/lib/sable-mode.scm
@@ -53,11 +53,11 @@
 
 
 (xml_register_id "-//SABLE//DTD SABLE speech mark up//EN"
-		(path-append libdir "Sable.v0_2.dtd")
+		(path-append xml_dtd_dir  "Sable.v0_2.dtd")
 		)
 
 (xml_register_id "-//SABLE//ENTITIES Added Latin 1 for SABLE//EN"
-		 (path-append libdir  "sable-latin.ent")
+		 (path-append xml_dtd_dir  "sable-latin.ent")
 		 )
 
 ;; (print (xml_registered_ids))
